sdk/azcore/CHANGELOG.md

```diff
@@ -1,13 +1,11 @@
 # Release History
 
-## 1.18.0-beta.1 (Unreleased)
+## 1.18.0 (2025-04-03)
 
 ### Features Added
 
 * Added `AccessToken.RefreshOn` and updated `BearerTokenPolicy` to consider nonzero values of it when deciding whether to request a new token
 
-### Breaking Changes
-
 ### Bugs Fixed
 
 * Removed hardcoded `azcore` replacement in the error type for `runtime.StartSpan()`.
```

Comment:
I think this and the `Other Changes` were reverted from `main` and moved to the feature branch?